plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -567,5 +573,39 @@ describe('pg-pool', () => {
         });
       });
     });
+
+    it('should generate `db.client.*` metrics (Promises-style)', async (...args) => {
+      const client = await pool.connect();
+
+      try {
+        const ret = await client.query('SELECT NOW()');
+        assert.ok(ret);
+      } finally {
+        client.release();
+      }
+
+      const { resourceMetrics, errors } = await metricReader.collect();
+      assert.deepEqual(
+        errors,
+        [],
+        'expected no errors from the callback during metric collection'
+      );
+
+      // We just test the expected metric *names* here. The particulars of the
+      // metric values are already tested in other test cases.
+      const metrics = resourceMetrics.scopeMetrics[0].metrics;
+      assert.strictEqual(
+        metrics[0].descriptor.name,
+        'db.client.operation.duration'
```

Comment:
nit: can you use the variable here? 
I made a similar update on https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2475/files#diff-682483846290c7e14fba934e5f5effaaa2ff8e84e94f511300af6d0120a59251R53

Reply:
done in commit 2120edf4